src/hints.rs

```diff
@@ -64,3 +64,20 @@ pub fn skip(
 ) -> Result<(), VirtualMachineError> {
 	Err(VirtualMachineError::CustomHint("skip".to_string()))
 }
+
+pub fn mock_call(
+	vm: &mut VirtualMachine,
+	exec_scopes: &mut ExecutionScopes,
+	ids_data: &HashMap<String, HintReference>,
+	ap_tracking: &ApTracking,
+	_constants: &HashMap<String, BigInt>,
+) -> Result<(), VirtualMachineError> {
+	let func_to_mock = get_ptr_from_var_name("func_to_mock", vm, ids_data, ap_tracking)?;
+	let mock_ret_value = get_integer_from_var_name("mock_ret_value", vm, ids_data, ap_tracking)?;
+	exec_scopes.assign_or_update_variable(
+		&format!("mock_call_{}", func_to_mock.offset),
```

Comment:
Put it in a sub hashmap with key `mock_calls`

Reply:
done